app/models/game.rb

```diff
@@ -1,4 +1,63 @@
 class Game < ActiveRecord::Base
   belongs_to :users
   has_many :pieces
+  after_create :initialize_board!
+
+  def initialize_board!
+    make_pawns!
+    make_rooks!
+    make_bishops!
+    make_knights!
+    make_kings!
+    make_queens!
+  end
+
+  def make_pawns!
+    8.times do |n|
+        pieces << Pawn.create(
+            color: 'white', 
+            x_cord: n,
+            y_cord: 1
+            )
+    end
+    #
+    8.times do |n|
+        pieces << Pawn.create(
+            color: 'black', 
+            x_cord: n,
+            y_cord: 6
+            )
+    end
+  end
+
+  def make_rooks!
+    pieces << Rook.create(color: 'white', x_cord: 0, y_cord: 0)
+    pieces << Rook.create(color: 'white', x_cord: 7, y_cord: 0)
```

Comment:
Just double checked in the rail console. I made a sample game with `Game.create()`. I then checked the game_ids of both the Game and the Pieces and they're both the same. However, if I remove `pieces <<` and repeat the process in the rails console, then their game_ids don't match.